lib/utils/uniteSets.cjs

```diff
@@ -6,6 +6,7 @@
  * Unite two or more sets
  *
  * @param {Iterable<string>[]} args
+ * @see {@link https://github.com/microsoft/TypeScript/issues/57228|GitHub}
  */
 function uniteSets(...args) {
```

Comment:
@ybiquitous

Should we add https://github.com/es-shims/Set.prototype.union?

Reply:
No, I don't think we need to add the polyfill package for this tiny utility function.
The package has many dependencies, which increase Stylelint's download size.

See https://pkg-size.dev/set.prototype.union
<img width="293" alt="image" src="https://github.com/stylelint/stylelint/assets/473530/efc2a1fd-790d-4746-bc70-abe81d17fecf">